app/tools/android/make_apk.py

```diff
@@ -903,21 +903,17 @@ def main(argv):
     CleanDir(xpk_temp_dir)
     ParseXPK(options, xpk_temp_dir)
 
+  manifest = None
```

Comment:
You don't need this here; Python's scoping works so that you can move this line to the `else` clause below.